README.md

```diff
@@ -22,6 +22,12 @@
 - Please read _all_ the links included:
 - Take a look at [our **Guidebook**](https://bwtc-handbook.notion.site/BWTC-Handbook-71256770d53f46439aecd989b07d1533) for all your (mostly) non-code related needs!
 
+### GitHub
+
+- Please create a Github account using your **work** email
+- Once an account has been created, open your terminal to configure your Git username/email.
+  <a href="https://support.atlassian.com/bitbucket-cloud/docs/configure-your-dvcs-username-for-commits/">![Gitflow diagram](./git-config.png)</a>
```

Comment:
While `a` tags and html elements will definitely work within GH docs, lets keep that markdown standard for now. Its what we use for almost all of our other docs. I believe this should be equivalent: 

```suggestion
    Once an account has been created, open your terminal to configure your Git username/email. [Gitflow diagram](git-config.png)
  

```

Reply:
Also how do you feel about making the GitHub/git config setup its own section within our documentation and removing the png? That way, we can stay relevant and not need to update the png if/when GH changes how they want us to set up our config.

Reply:
@michaelachrisco I believe this is what you wanted

```suggestion
  [![Gitflow diagram](./git-config.png)</a>](https://support.atlassian.com/bitbucket-cloud/docs/configure-your-dvcs-username-for-commits/)
```